looprpc/client.proto

```diff
@@ -740,6 +746,16 @@ message LsatToken {
     string storage_name = 8;
 }
 
+message GetInfoRequest {
+}
+
+message InfoParameters {
+    /*
+    The current daemon version.
+     */
+    string version = 1;
```

Comment:
not sure what we're gonna add here, but keep in mind that currently we can get autoloop related parameters from the `GetLiquidityParams` call. Ideally we wouldn't want to overlap.

Perhaps a summary of swaps / swapped amounts / swap costs would fit here?

Reply:
Good point, the initial list of params comes from the related issue https://github.com/lightninglabs/loop/issues/583.
It contains your suggestions... swaps/amounts/active swaps.